### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by paulkannan**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by paulkannan** [comment]
could you pls tell me whats the exact error, since this is a cdk code, why do i require template.yml?

File: cdk-s3-lambda-textract-comprehend-s3/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "title": "Serverless Document Processing and Identification using Textract and Comprehend with CDK",
```
Comment (boyney123) [inline cdk-s3-lambda-textract-comprehend-s3/example-pattern.json:2]:
```suggestion
  "title": "Document processing and identification using Textract and Comprehend",
```

File: cdk-s3-lambda-textract-comprehend-s3/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "description": "To analyse and identify the uploaded document whether it is a valid Government ID. Implemented in CDK",
```
Comment (boyney123) [inline cdk-s3-lambda-textract-comprehend-s3/example-pattern.json:3]:
```suggestion
  "description": "Analyze and identify documents whether it is a valid Government ID. Implemented in CDK",
```

File: cdk-s3-lambda-textract-comprehend-s3/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "language": "Python",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This sample project demonstrates how to analyze and identify the uploaded document as a valid government issued ID. Implemented with CDK (Python)"
```
Comment (boyney123) [inline cdk-s3-lambda-textract-comprehend-s3/example-pattern.json:10]: This is a copy of the description, @paulkannan can you give us more details on how the pattern works? Nothing too much, but dive deeper into the pattern slighty.

**Comment by boyney123** [comment]
Thanks @paulkannan , I removed the template for you, also can you add some comments to the `how it works` just dive slightly deeper into the pattern.

**Comment by paulkannan** [comment]
Hi @boyney123 , thanks for the changes you have made. I will delete this repo as Im further enhancing the model by use of Rekognition and I will publish this as a How To instead of pattern because Im planning to cover Intelligent Document Processing by using serverless. Once again thanks for your support

**Pull request closed without merging**